tests/func-tests/virtual_machines_test.go

```diff
@@ -59,7 +59,8 @@ func verifyVMIRunning(client kubecli.KubevirtClient, vmiName string) *kubevirtco
 		var err error
 		vmi, err = client.VirtualMachineInstance(kvtutil.NamespaceTestDefault).Get(context.Background(), vmiName, &k8smetav1.GetOptions{})
 		g.Expect(err).ToNot(HaveOccurred())
-		fmt.Fprintf(GinkgoWriter, "VMI's status.phase: %s", vmi.Status.Phase)
+		GinkgoWriter.Printf("VMI's status.phase: %s\n", vmi.Status.Phase)
+		GinkgoWriter.Printf("Reason: %s\n\n", vmi.Status.Reason)
```

Comment:
These aren't really a part of this PR, but I have added this because 1.27 tests keep failing here as pointed out in earlier comments.

Reply:
https://github.com/kubevirt/hyperconverged-cluster-operator/pull/2310#issuecomment-1552851140

https://github.com/kubevirt/hyperconverged-cluster-operator/pull/2310#issuecomment-1552694525

Reply:
Still seeing the failure, but don't see anything in the Reason. :) 
```
 [rfe_id:273][crit:critical][vendor:cnv-qe@redhat.com][level:system]Virtual Machine [test_id:5696] should create, verify and delete VMIs
/home/prow/go/src/github.com/kubevirt/hyperconverged-cluster-operator/tests/func-tests/virtual_machines_test.go:35
  Run 1/5
  STEP: Creating VMI... @ 05/18/23 11:47:13.546
  STEP: Verifying VMI is running @ 05/18/23 11:47:13.57
  VMI's status.phase: Pending
  Reason: 
  VMI's status.phase: Failed
  Reason: 
  VMI's status.phase: Failed
  Reason: 
...
...
[FAILED] in [It] - /home/prow/go/src/github.com/kubevirt/hyperconverged-cluster-operator/tests/func-tests/virtual_machines_test.go:39 @ 05/18/23 11:57:13.571
• [FAILED] [600.224 seconds]
[rfe_id:273][crit:critical][vendor:cnv-qe@redhat.com][level:system]Virtual Machine [It] [test_id:5696] should create, verify and delete VMIs
/home/prow/go/src/github.com/kubevirt/hyperconverged-cluster-operator/tests/func-tests/virtual_machines_test.go:35 
```